evidence/rossbulat/0002-rossbulat-retain.md

```diff
@@ -0,0 +1,88 @@
+- Beyond this focus, Developer Console identifies the shortfalls of JS Apps and demonstrates tangible ways it can be improved (these are demonstrated in the presentation).
+- I also studied RFC 0097 in detail and understand the unbonding queue mechanism, and how to implement this in the Polkadot SDK runtime.
+
+My work is relevant for the specific Manifesto requirements:
+- the Polkadot business-logic (aka the 'runtime');
+- pallets utilized by the Polkadot (Main) Network and its system chains;
+- runtime and host APIs
+- user-interface code required to practically execute upgrades to the Polkadot (Main) Network
+
+Polkadot SDK PRs:
+
+- [RFC-0097 Implementation to Decrease Unbonding Time](https://github.com/paritytech/polkadot-sdk/pull/5449)
+- [Remove staking controller account logic from pallets & runtimes](https://github.com/paritytech/polkadot-sdk/pull/5715)
+
+Social:
+
+- [Announcement presentation of Polkadot Developer Console initiative](https://x.com/rossbulat/status/1825967285896380659)
+- Continue to engage with staking related activities.
+
+
+Documentation / Education:
+
+- Launched and open sourced Polkadot Cloud docs. [website](https://docs.polkadot.cloud/) / [source code](https://github.com/polkadot-cloud/docs).
+
+
+Staking Dashboard PRs:
+- [feat: Check if Polkadot app is at least generic version](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2191) 
+- [feat(refactor): People chain on Polkadot - disable identities on Relay Chain](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2192)
+- [feat(refactor): Abstract state bootstrapping & subscriptions from Api](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2193)
+- [feat(refactor): Use merkelise-metadata for Ledger signing, discontinue metadata service](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2195)
+- [feat(refactor): @substrate/connect updates](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2196)
+- [feat(refactor): Remove pre-paged rewards deprecation logic](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2197)
+- [feat: Activate People Chain and re-enable identities](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2198)
+- [fix: people status on identity sync](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2236)
+
+
+Developer Console:
+- Launch of Public Alpha and positioning to be an eventual alternative to Polkadot JS Apps.
+- Polkadot Developer Console [app](http://console.polkadot.cloud) / [source code](https://github.com/polkadot-cloud/polkadot-developer-console).
+
+There are too many PRs / commits to list here, so the evidence instead points to the source code of Developer Console that was rapidly iterated leading up to its public alpha release, as well as the public presentation.
+
+## Voting record
```

Comment:
The voting record is a mandatory part of the evidence report, so it needs to be reinstated here.
On the row for "Rank 1", please add info about the referenda you voted on. 
FYI: As a rank 1, you were eligible to vote on any of the referenda listed [here](https://collectives.subsquare.io/fellowship/tracks/1) over the past 2 months.

Reply:
I have added an empty table and note below it.

Reply:
Thank you.
As a general rule, it is best to keep the template (and the info about thresholds) as it is to avoid confusion for reviewers. So, any comment about your voting record should go into the table that is part of the template.
Here is an example of what it could look like: 
![image](https://github.com/user-attachments/assets/d0a14ebb-d1cb-416e-9ac5-3f29568ac05c)

Reply:
Thank you, I reformatted it as per the screenshot.